### ISSUE TITLE
About the crop size in COCO

### ISSUE THREAD

**Issue by Reagan1311** [Jia-Research-Lab/PFENet#14]
Hi, I found in your paper the cropping size is 473 x 473 in training with a learning rate of 0.005. (COCO)
However, the cropping size in coco config is 641 x 641 with a learning rate of 0.02.
Just want to know what's the exact cropping size and learning rate in coco dataset, thank you!

**Comment by tianzhuotao** [comment]
@Reagan1311  For fairness, please use the config files provided in this repo to have optimal performance on the ground truth images that do not be altered.
